massgov/mayflower
# DP-12883: run npm audit fix to remove vulnerabilities

## Description
Update vulnerabilities using npm audit fix

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-12883)
- [Github issue](https://github.com/massgov/mayflower/compare/patternlab/DP-12883-update-mf-patternlab?expand=1)

## Steps to Test
1. view the styleguide and make sure nothing looks incorrect while using this branch.

## Screenshots
N/A


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
